assets/game_projects/fighter/src/main.py

```diff
@@ -37,6 +37,11 @@ def _start(self) -> None:
         self.fight_state = FightState()
         self.frame_counter = 0
         self.input_buffers = []
+        self.x_intensity_max = 0.0 #the x shake the camera has
```

Comment:
Can you put this logic in a separate class.  Maybe like `CameraShaker`.  I'm removing stuff from main as it's getting cluttered.